src/com/lftechnology/java/training/krishna/employeemanagement/domain/Employee.java

```diff
@@ -0,0 +1,139 @@
+
+package com.lftechnology.java.training.krishna.employeemanagement.domain;
+
+/**
+ * This program store employee information
+ * 
+ * @author Krishna Timilsina <krishnatimilsina@lftechnology.com>
+ * @version 1.0
+ */
+public class Employee extends User {
+
+	private String fullname;
+	private String department;
+	private String address;
+	private String role;
+
+	
+	public Employee() {
+
+	}
+
+	public Employee(
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Constructor has 10 parameters, which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00107)